src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -234,7 +234,18 @@ export class DynamicFacet extends Component implements IDynamicFacet {
      *
      * Setting this option to a higher value may enhance the accuracy of facet value counts at the cost of slower query performance.
      */
-    injectionDepth: ComponentOptions.buildNumberOption({ defaultValue: 1000, min: 0 })
+    injectionDepth: ComponentOptions.buildNumberOption({ defaultValue: 1000, min: 0 }),
+
+    /**
+     * TODO: document option
```

Comment:
> Whether to exclude folded result parents when estimating result counts for facet values.
>
> See also the [`Folding`]{@link folding} and [`FoldingForThread`]{@link FoldingForThread} components.
>
> **Default:** `true` if folded results are requested;`false` otherwise.